src/redux/reducers/componentConfig/updateConfig.js

```diff
@@ -1,16 +1,21 @@
 import _ from 'lodash';
 
 const updateConfig = (state, action) => {
-  const { plotUuid, configChange } = action.payload;
+  const { plotUuid, configChange = null, dataChange = null } = action.payload;
+  let newConfig = state[plotUuid].config;
+  if (configChange) {
```

Comment:
what does this if statement evaluate to when `configChange` is null?

Reply:
If it is null it doesn't merge `newConfig` with the previous config.

Reply:
fair enough